lookout/style/format/analyzer.py

```diff
@@ -87,8 +88,8 @@ class FormatAnalyzer(Analyzer):
             "trainable_rules": {
                 "prune_branches_algorithms": ["reduced-error"],
                 "top_down_greedy_budget": [False, .5],
-                "prune_attributes": True,
-                "uncertain_attributes": True,
+                "prune_attributes": False,
+                "uncertain_attributes": False,
```

Comment:
This should be True, becase False actually turns this pruning on.

Reply:
oops, debug stuff.